Prox/Prox/Map View/MapViewController.swift

```diff
@@ -77,31 +81,50 @@ class MapViewController: UIViewController {
     init(enabledFilters: Set<PlaceFilter>) {
         self.enabledFilters = enabledFilters
         super.init(nibName: nil, bundle: nil)
+
+        self.modalPresentationStyle = .overCurrentContext
+        self.transitioningDelegate = self
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) not implemented")
     }
 
     override func viewDidLoad() {
-        view.backgroundColor = Colors.mapViewBackgroundColor
-
         let closeButton = UIButton()
         closeButton.setImage(#imageLiteral(resourceName: "button_dismiss"), for: .normal)
         closeButton.addTarget(self, action: #selector(closeWithButton), for: .touchUpInside)
 
-        for subview in [self.mapView, searchButton, self.placeFooter, closeButton] as [UIView] {
+        let container = UIView()
+        container.backgroundColor = Colors.mapViewBackgroundColor
+
+        for subview in [container, mapView, searchButton, placeFooter, closeButton] as [UIView] {
             view.addSubview(subview)
         }
 
+        container.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+
+            showMapConstraints = [
+                make.top.equalToSuperview().constraint,
+                make.bottom.equalTo(bottomLayoutGuide.snp.top).constraint,
```

Comment:
Just copied from [what we were using before](https://github.com/mozilla-mobile/prox/pull/630/files#diff-8364cdee0b6d84d4da354f4b18cc16b5L116). I'll just change this to bottom.